cli/cmd/manifestUpdate.go

```diff
@@ -51,9 +51,15 @@ func cliManifestUpdate(manifestName string, host string, clCertFile string, clKe
 
 	// Set rootCA for connection to coordinator
 	certPool := x509.NewCertPool()
-	if ok := certPool.AppendCertsFromPEM([]byte(caCert)); !ok {
+	if ok := certPool.AppendCertsFromPEM([]byte(caCert[0].Bytes)); !ok {
 		return errors.New("Failed to parse certificate")
 	}
+	// Add intermediate cert if applicable
+	if len(caCert) > 1 {
+		if ok := certPool.AppendCertsFromPEM([]byte(caCert[1].Bytes)); !ok {
+			return errors.New("Failed to parse certificate")
+		}
+	}
```

Comment:
This is a bit confusing. caCert[0] is the intermediate, caCert[1] is the root cert (in case of a present intermediate).
Please change the comment accordingly.